AppCenter/AppCenterTests/MSLogDBStorageTests.m

```diff
@@ -1061,6 +1061,9 @@ - (void)testErrorDeletingOldestLog {
 
 - (void)testCreateFromLatestSchema {
 
+  // If
+  [self.storageTestUtil deleteDatabase];
```

Comment:
`[self.storageTestUtil deleteDatabase]` is already called in [`(void)tearDown`](https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1499/files#diff-ae42d5f18ad4a9fc0a4426cbbb33eb98R57), why duplicate here?